lib/network/kmip/Message.js

```diff
@@ -0,0 +1,52 @@
+'use strict'; // eslint-disable-line
+
+const assert = require('assert');
+
+
+function _lookup(decodedTTLV, path) {
+    const xpath = path.split('/');
+    const obj = decodedTTLV;
+    let res = [];
+    for (let current = xpath.shift(); current; current = xpath.shift()) {
+        assert(Array.isArray(obj));
+        for (let i = 0; i < obj.length; ++i) {
+            const cell = obj[i];
+            if (cell[current]) {
+                if (xpath.length === 0) {
+                    res.push(cell[current].value);
+                } else {
+                    const intermediate =
+                          _lookup(cell[current].value, xpath.join('/'));
+                    res = res.concat(intermediate);
+                }
+            }
+        }
+    }
+    return res;
+}
+
+class Message {
+    /**
+     * Construct a new abstract Message
+     * @param {Object} content - the content  of the message
+     */
+    constructor(content) {
+        if (!new.target) {
```

Comment:
this check seems to be redundant since the `new` keyword is required to call the constructor

Reply:
Oh yes!